scene/gui/control.cpp

```diff
@@ -2286,7 +2286,7 @@ void Control::set_theme(const Ref<Theme> &p_theme) {
 	}
 
 	if (data.theme.is_valid()) {
-		data.theme->connect("changed", this, "_theme_changed");
+		data.theme->connect("changed", this, "_theme_changed", Vector<Variant>(), CONNECT_DEFERRED);
```

Comment:
I think you can use `varray()` for the default parameter, that's what other such calls use.

Reply:
I saw both in this file, but I changed it.